### PR TITLE
CLI-WG project description

### PR DESCRIPTION
**Pull request by spacekookie**
Maybe look over it? Definitely not done yet though!

### CODE REVIEW

File: templates/projects.hbs [hunk 1]
```diff
@@ -56,4 +56,17 @@
       </div>
     </div>
   </div>
+  <div class="project">
+    <div class="row">
+      <div class="description">
+        <h3>clap-rs</h3>
+        <p><a href="http://clap.rs/">clap</a> is a fast and configurable argument parsing library for Rust. It is the de-facto standard in the crates ecosystem for handling provided arguments when creating commandline applications. </p>
```
[the remark this answers is not in the document]
Comment (kbknapp) [reply templates/projects.hbs]: I'm not super great talking about myself, but something like:

1. good candidates for this project would be people who are excited about learning about building robust CLI applications with all the modern niceties and improving CLI UX

For 2, forgive me if I'm catching up late, but do we need to already have a good project for them to build? If no, here is a generic statement I'd use:

2. participants in the project will focus on CLI design including detailed UX, building robust CLI applications in Rust while avoiding common mistakes and performance pitfalls, learn how to structure a CLI application to as to be maintainable and extensible, and finally learn common considerations when packaging and distributing an application.

If we need an actual project in mind, I'd need to think of one (and I have a few ideas off the top of my head) :stuck_out_tongue_winking_eye: 

Also, here's what I sent to @spacekookie recently:

> Kevin is the creator of clap and a member of the CLI Working Group. He is fascinated by all things command line and passionate about building, and teaching others how to build a command line world with better UX and more consistency. 

[Along with this image](https://clap.rs/wp-content/uploads/2018/04/01505921707661abc.jpg)

**Comment by spacekookie** [comment]
There, I went ahead and changed the bio+picture. And tried to boil down some notes you wrote into a second paragraph. @ashleygwilliams @kbknapp What do you think?

**Comment by ashleygwilliams** [comment]
@spacekookie this is very kind of you thank you! so- the one thing i would say is... let's rebase this and see how it looks next to the other CLI project? i'm worried it might be a bit confusing and maybe we should try to combine them somehow- or distinguish them more? either way i think it's a pretty solid start and except for needing to rebase this and then mitigate the fact that there are now 2 projects (which is... awesome), so yeah once we sort that i think we can merge and deploy!

File: templates/projects.hbs (hunk 1 above)
[the remark this answers is not in the document]
Comment (ashleygwilliams) [reply templates/projects.hbs]: if you have an actual project in mind i can tell u from looking at the applications we've already gotten that people really love the more specific projects :) but it's not necessary- really up to you!

**Comment by spacekookie** [comment]
@ashleygwilliams Oh okay, I didn't see that there was a second project already :) Are you gonna do the rebasing of branches then? I could imagine it looking good either as two projects or merged into one (with highlights for different projects)

**Comment by ashleygwilliams** [comment]
@spacekookie up to you! if you wanna play with it and see what you think is best i am very happy to leave it in your capable hands. if you don't want to and/or are too busy just lemme know and i can work it out :)

**Comment by ashleygwilliams** [comment]
@kbknapp lemme know what you think! if you approve, i'll merge and deploy and we can make a special announcement that we're adding your project- i'm going to add one more project as well (an internationalization project on the rust-lang.org proj) so we'll add 2 more :D

**Comment by kbknapp** [comment]
I'm good with it! Also, if you wanted to combine myself and @Aaronepower I'm fine with that too :wink:

**Pull request merged**